test/clients/messaging-api/Api/MessagingApiApiTest.php

```diff
@@ -276,6 +297,11 @@ public function testListCoupon(): void
         $start = 'startToken';
         $limit = 10;
         $contentType = 'application/json';
+        $expectedQuery = [
+            'status' => $status,
+            'start'  => $start,
+            'limit'  => (string)$limit,
+        ];
         $expectedRequestBody = <<<JSON
```

Comment:
I’m sorry for adding this code, but `expectedRequestBody` is not needed here, so please delete it.
Everything else is LGTM.